recipes/libnl-devel-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: libnl-devel-cos6-x86_64
+  version: 1.1.4
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libnl-devel-1.1.4-2.el6.x86_64.rpm
+    sha256: 6c1749d9f8e9cbe31c84067a7da5f9861c2cd61a0df9b83107b6d045f24a766a
+    folder: binary
+
+build:
+  number: 0
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - libnl-cos6-x86_64 ==1.1.4
+  run:
+    - libnl-cos6-x86_64 ==1.1.4
+
+test:
+  commands:
+    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
+  requires:
+    - zlib
+
+about:
+  home: http://www.infradead.org/~tgr/libnl/
+  license: LGPLv2
```

Comment:
We are not enforcing nor it is required but it would be nice to use the SPDX strings for consistence. Would you mind opening an issue in conda-build ?

Reply:
Sure have raised issue ( https://github.com/conda/conda-build/issues/3589 ).